docs/development/codeguide.rst

```diff
@@ -165,17 +157,6 @@ Coding Style/Conventions
     need to be used in affiliated packages. In particular, the set of ruff_ checks is
     not required for affiliated packages.
 
-  * Alternately, you can manually check and fix your changes by running the
-    following `tox <https://tox.readthedocs.io/>`__ command::
-
-      tox -e codestyle
```

Comment:
It seemed redundant to include both this and the note below. I chose to keep the note.